### DISCUSSION TITLE
Macbook Pro not tracking

### DISCUSSION THREAD

**Discussion by stevencoutts** [ESPresense/ESPresense#169] [General]
Hi all

I'm new to this so apologies for any daft questions :)

I've set up 3 sensors so far and Apple Watch and iPhone detection seems to be working quite well.

I've tried to track my Macbook Pro and it sometimes gets seen by the sensors but mostly it doesn't seem to update.

Any idea?

**Comment by ndbroadbent** [comment]
@siiiilky Were you able to get this working?

I'm also interested in tracking our Macbooks if we move them around the house and work in different rooms. I seem to be struggling to figure out which device is my Macbook (if any). I'd also like to track my wife's Macbook. Are we able to use the IRK keys for Macbooks as well? (It's working great for my Apple Watch and iPhone.)
